hal/include/bmp180_device.h

```diff
@@ -0,0 +1,179 @@
+#ifndef BMP180_DEVICE_H_
+#define BMP180_DEVICE_H_
+
+#include <thread>
```

Comment:
Do not include some header that is not referenced in the header file. Because when another class is referencing this class, will increase the time for processing headers. Put the header file to the source file (CPP file).